modules/administration-guide/partials/proc_applying-the-github-oauth-app-secret.adoc

```diff
@@ -0,0 +1,58 @@
+:_content-type: PROCEDURE
+:description: Applying the GitHub OAuth App Secret
+:keywords: applying-the-github-oauth-app-secret, apply-the-github-oauth-app-secret, apply-github-oauth-app-secret, apply-secret, applying-secret, apply-a-secret, applying-a-secret
+:navtitle: Applying the GitHub OAuth App Secret
+// :page-aliases:
+
+[id="applying-the-github-oauth-app-secret_{context}"]
+= Applying the GitHub OAuth App Secret
+
+Prepare and apply the GitHub OAuth App Secret as follows:
```

Comment:
Minimalism: `Prepare and apply the GitHub OAuth App Secret`

Reply:
Thank you, Jake. Done.